msal/application.py

```diff
@@ -1385,10 +1404,13 @@ def _acquire_token_silent_from_cache_and_possibly_refresh_it(
                     claims=_merge_claims_challenge_and_capabilities(
                         self._client_capabilities, claims_challenge),
                     correlation_id=correlation_id,
+                    auth_scheme=auth_scheme,
                     **data)
                 if response:  # The broker provided a decisive outcome, so we use it
                     return self._process_broker_response(response, scopes, data)
 
+            if auth_scheme:
+                raise ValueError("auth_scheme is currently only available from broker")
```

Comment:
Consider adding more details like how to enable the broker redirect uri, or an aka.ms link